### PR TITLE
 BUGFIX: BB-1438 Fix grouping deactivation when column attributes used

### PR DESCRIPTION
**Pull request by jkittler**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3434/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2730/ : FAILURE in 4m 58s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3435/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2731/ : FAILURE in 1m 59s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3436/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2733/ : FAILURE in 10m 44s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3441/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2742/ : SUCCESS in 5m 34s

**Comment by jkittler** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-847.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3442/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/881/ : SUCCESS in 32m 08s

**Comment by jkittler** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3447/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2747/ : SUCCESS in 4m 02s

File: src/components/core/PivotTable.tsx [hunk 1]
```diff
@@ -901,8 +906,12 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
             { attributes: [], all: [] }
         );
 
-        return sortedColumnIndexes.attributes[0] === 0 && sortedColumnIndexes.all.length === 1 ||
-            sortedColumnIndexes.all.length === 0;
+        const i = sortedColumnIndexes.attributes[0] === 0
```
Comment (yanlipnican) [inline src/components/core/PivotTable.tsx]: Remove variable `i`
And i think the condition is bit messy and i don't understand it. I would split them to some logical blocks like
```
isFirstConditionTrue = sortedColumnIndexes.attributes[0] === 0;
isSecondConditionTrue = ...;
return isFirstConditionTrue || isSecondConditionTrue
```

**Comment by yanlipnican** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3451/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2750/ : SUCCESS in 3m 30s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3453/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2754/ : SUCCESS in 4m 34s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/758/ : SUCCESS in 31s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2767/ : SUCCESS in 8m 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1468/ : SUCCESS in 33m 56s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/736/ : SUCCESS in 28m 42s

**Pull request merged**